docs/guide/dev/features/easyimage_integration/README.md

```diff
@@ -0,0 +1,94 @@
+---
+category: integration
+order: 40
+url: guide/dev_easyimage_integration
+menu-title: Easy Image
+meta-title-short: Easy Image
+---
+<!--
+Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Easy Image Integration
+
+## Upload integration
+
+Easy Image uses [Cloud Services](https://ckeditor.com/ckeditor-cloud-services/) backend by default, which needs to be configured before usage of Easy Image plugin. To configure Cloud Service provider, please refer to official [Cloud Services documentation](https://docs.ckeditor.com/cs/latest/index.html).
+
+## Responsiveness
+
+Output produced by Cloud Services is converted to include several versions of the uploaded images, which have different widths and are suitable for different types of devices. It's handled by `[srcset]` attribute for `img` tag and looks similar to the markup presented below:
+
+```html
+<figure class="easyimage easyimage-full">
+	<img alt="" width="1026" srcset="https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_110 110w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_220 220w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_330 330w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_440 440w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_550 550w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_660 660w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_770 770w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_880 880w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_990 990w, https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png/w_1026 1026w" sizes="100vw" src="https://cdn.cke-cs.com/f0pqzdtf0yRhaX1FymZU/images/b21777b6cdb9359c2ad629a79878c2a643a958964d0096de_photo.png" />
```

Comment:
Let's provide an `alt`, let's show that we like to follow good accessibility practices.